orthosie/settings.py

```diff
@@ -104,7 +104,11 @@
 }
 
 TAX = .075
+
 PRINTER = '/dev/null'
+# PRINTER = {'spool':'/dev/null', 'driver':'orthosie', 'interface':''}
```

Comment:
Uncomment this line so it becomes the default configuration. Remove the other two PRINTER config lines.